### DISCUSSION TITLE
Variable Inheritance

### DISCUSSION THREAD

**Discussion by grandon-lin** [Dynatrace/dynatrace-configuration-as-code#1040] [Q&A]
Hi,
Is there any possibility that we can specify a variable in a config yaml file, and this variable can be automatically used without specifying it for each config? For example, in a config yaml file, all my configs use variable mgntZone: abc1, but instead of putting this var in each config, is it possible for each config to automatically assume this mgntZone: abc1 is used?

Thanks,
Grandon

**Comment by warber** [comment]
Hi @grandon-lin 
Currently, this is not possible. There was the idea of actually introducing the possibility of adding "_global_" variables defined in the manifest that can be used everywhere. We are considering adding this feature, especially because you are not the first person requesting that...

**Reply by grandon-lin** [reply]
I see. Thanks very much.
